src/styles.css

```diff
@@ -70,14 +70,22 @@
       top: 8px;
       left: 8px; }
     .legendables .tick:hover:after {
-      border-color: #22A7F0; }
+      border-color: #22a7f0; }
+  .legendables .sort {
+    position: absolute;
+    right: 14px;
+    top: 4px;
+    fill: #a7a7a7;
+    cursor: pointer; }
+    .legendables .sort:hover {
+      fill: #22a7f0; }
   .legendables .header {
     color: #a7a7a7;
     font-weight: bold;
     margin: 0 4px 0 4px;
     position: relative;
     height: 24px;
-    padding-right: 24px;
+    padding-right: 32px;
```

Comment:
Modified padding here and below to make room for new button

Reply:
This still overflows into the button a bit, a little margin (8px below) for the `.legendables .title-text` below helps alleviate that. 
<img width="214" alt="Screen Shot 2024-07-29 at 1 56 16 PM" src="https://github.com/user-attachments/assets/c4d91213-0abb-449a-af8b-5e0257be5302">
<img width="216" alt="Screen Shot 2024-07-29 at 1 56 21 PM" src="https://github.com/user-attachments/assets/08c294fe-3190-4691-a021-90502317cf92">

Reply:
Maybe more than 8 actually, if the text is just at the limit it can overlap pretty far
<img width="213" alt="Screen Shot 2024-07-29 at 2 19 03 PM" src="https://github.com/user-attachments/assets/a51189e6-4253-45ea-949c-12857ee87835">

Reply:
It's weird you're seeing that, because I don't see that at all - and honestly that just looks like the css changes I did didn't make it through...wonder if something is off with the revisions to the build system I did.  Let me dig in.

![image](https://github.com/user-attachments/assets/ed351ceb-0ded-4959-8218-0f7a53b60973)

Reply:
@johallar Do you mind trying again?  I updated the immerse PR with an updated reference to legendables (since it's a dep of charting, it was hash-linked in package-lock).  You will have to blow away `node_modules` and do `npm install` (`npm ci` didn't catch the updated legendables dep when I tried).  Let me know if you still see the issue after that.  For reference, here's after I finished doing all this:
![image](https://github.com/user-attachments/assets/70f9cbf3-443b-4007-9afd-c3f3f9d916bd)

Reply:
Trying again now